library_generation/README.md

```diff
@@ -122,6 +122,24 @@ The default value is `true`.
 
 Use `--rest_numeric_enums` to specify the value.
 
+### gapic_yaml (optional)
+One of GAPIC options passed to the generator.
+The default value is an empty string.
+
+Use `--gapic_yaml` to specify the value.
+
+### grpc_service_config (optional)
```

Comment:
Let's call it `service_config` as it is not specific to gRPC.

Reply:
done.